libraries/AC_PrecLand/PosVelEKF.h

```diff
@@ -1,18 +1,46 @@
 #pragma once
 
+/*
+* This class implements a simple 1-D Extended Kalman Filter to estimate the Relative body frame postion of the lading target and its relative velocity
+* position and velocity of the target is predicted using delta velocity
+* The predictions are corrected periodically using the landing target sensor(or camera)
+*/
 class PosVelEKF {
 public:
+    // Initialize the covariance and state matrix
+    // This is called when the landing target is located for the first time or it was lost, then relocated
     void init(float pos, float posVar, float vel, float velVar);
+
+    // This functions runs the Prediction Step of the EKF
+    // This is called at 400 hz
     void predict(float dt, float dVel, float dVelNoise);
+
+    // fuse the new sensor measurement into the EKF calculations
+    // This is called whenever we have a new measurement available
     void fusePos(float pos, float posVar);
-    void fuseVel(float vel, float velVar);
```

Comment:
Removed this function entirely because we aren't fusing velocities yet.. will bring it back if and when we do that..